llvm/include/llvm/IR/RepoHashCalculator.h

```diff
@@ -185,6 +187,9 @@ class HashCalculator {
   // Hold the global object list which the function hash depenendent on.
   ticketmd::DependenciesType Dependencies;
 
+  // Hold the global variables which the function/variable hash contributed to.
+  ticketmd::ContributionsType Contributions;
```

Comment:
Can I just check that the "contributions" are what we were referring to as "reverse dependencies"?

Reply:
Yes, should I rename to "ReverseDependencies"?